azure-ml-deployment/DeployModel.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
```

Comment:
It would be better to converge to using one common deployment method/mechanism. For instance, the Lambda Java uses gradle, Lambda Python uses a Makefile, and this Azure uses a shell script.

Reply:
Right now the only way to do the deployment is using the Azure CLI.

Reply:
https://docs.microsoft.com/en-us/azure/machine-learning/service/tutorial-deploy-models-with-aml

is there any way we can implement via python script then? As it is, this does not really mesh nicely with our ultimate goal which is to have 1-click deployment in dai no?

Reply:
It should not matter what it uses under the hood as long as we create a common interfaces for all of our deployments. Ultimately we will need to deploy from a python script, so agree with Nick, a python wrapper would be nice.

Reply:
I think python would be idea... we will really not want to add dependency on azure cli to dai